src/tools/actors/ActorExecutionTrace.java

```diff
@@ -83,8 +79,8 @@ public static void recordMainActor(final Actor mainActor,
   /**
    * Assign breakpoint for the actor that is the receiver of the message.
    */
-  public static void assignBreakpoint(final Breakpoint breakpoint, final Actor actor,
-      final BreakpointDataTrace breakpointTrace, final boolean receiver) {
+  /*public static void assignBreakpoint(final Breakpoint breakpoint, final Actor actor,
```

Comment:
Probably it is best to delete these lines, this approach was based on REME-D implementation to save the breakpoints in the LocalManager (BreakpointActor in our case) we still have to figure out if we need this link between BreakpointActor and breakpoints.